boards/arc/arduino_101_sss/arduino_101_sss.dts

```diff
@@ -66,4 +66,13 @@
 
 &spi1 {
 	status = "ok";
+
+	bmi160@1 {
```

Comment:
What does the `@1` mean in the `bmi160@1`? Is it slave number? Later, in _samples/sensor/bmi160/arduino_101_sss.overlay_, it's for `bmi160@0`.. where the spi-max-frequency is set to 88. Should it not be the same?

Also, I am not able to build the sample, because of the error: `arduino_101_sss.dts_compiled: ERROR (unit_address_vs_reg): /soc/spi@80010100/bmi160@0: node has a unit name, but no reg property`

Reply:
> What does the `@1` mean in the `bmi160@1`? Is it slave number? Later, in _samples/sensor/bmi160/arduino_101_sss.overlay_, it's for `bmi160@0` at line 9.. where the spi-max-frequency is set to 88. Should it not be the same?

The @1 is which chipselect the sensor is connected on.

I fixed the compile issue, so please try again.  I've gone w/the 640000 freq as per @tbursztyka comment.

Reply:
Yes, it builds fine now, thanks, also, have you tested this on arduino_101_sss?